gravitee-apim-repository/gravitee-apim-repository-jdbc/src/main/java/io/gravitee/repository/jdbc/management/JdbcApiKeyRepository.java

```diff
@@ -15,18 +15,23 @@
  */
 package io.gravitee.repository.jdbc.management;
 
-import static io.gravitee.repository.jdbc.common.AbstractJdbcRepositoryConfiguration.escapeReservedWord;
+import static org.springframework.util.CollectionUtils.*;
```

Comment:
Global note for all ApiKey repository modifications :
There was a lot of structural modification in ApiKeyRepository, and I think we have to add more test to ApiKeyRepositoryTest.

For example, the modification of `findById` is not tested.
We could add a test to ensure the table join is correct and subscriptions list is correctly retrieved